Blazor_Foundation_6_Test_Shared/Documentation/Containers/Callout.razor

```diff
@@ -1,6 +1,119 @@
 ﻿@page "/callout";
 
-<h3>Callout</h3>
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
+@using OpenCodeDev.Blazor.Foundation.Components.Controls;
+@using OpenCodeDev.Blazor.Foundation.Components.Containers;
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Callout</h2>
+        <p>Callouts combine panels and alerts from Foundation 5 into one generic container component.</p>
+
+        <h2>Basic</h2>
+        <p>A callout is just an element with a <Code>.callout</Code> class applied. You can put any kind of content inside.</p>
```

Comment:
do not refenrece class name like Foundation does... we are working in Blazor so you should reference a Blazor Component but never .XXX. Eg: `<Code Content="PEJ1dHRvbkdyb3VwPg=="></Code>` See Button Group-> Basics for full example. Do make class ref too...

Ends down to something like "You can put any kind of content inside the callout component by using  `<Code Content="PEJ1dHRvbkdyb3VwPg=="></Code>` " ....